src/pkg/utils/utils.go

```diff
@@ -0,0 +1,283 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/containers/toolbox/pkg/podman"
+	"github.com/shirou/gopsutil/host"
```

Comment:
Does `github.com/shirou/gopsutil/host` use `os-release`? Maybe we should use `github.com/acobaugh/osrelease` instead?

Reply:
Fine by me. `github.com/shirou/gopsutil/host` was the first thing that did what I needed but it may be unnecessarily big for our use case.